### DISCUSSION TITLE
Setting Z-offset via browser using the macros is not working

### DISCUSSION THREAD

**Discussion by manwald** [Guilouz/Klipper-Flsun-Speeder-Pad#68] [General]
Hey guys,

I converted my Speeder Pad with this amazing guide to the official Klipper-distribution - thanks a lot for this, Cyril!

I struggled a few days now, that setting the Z-offset using my browser, "PROBE_CALIBRATE" via the console and the "MOVE TO Z0"-macro for testing after setting the Z-offset was not working. I always ended up with the nozzle about 2mm above the printbed, even when I set Z-offset correctly, "ACCEPT" and "SAVE_CONFIG" everything.

Today I was able to set the correct Z-offset - using the touchscreen of the Speeder Pad. I used the menu-buttons for it and everything worked flawlessly.

It seems that there is something wrong when you use the browser, console and macros for setting the Z-offset, as it's not beeing saved correctly.

Best regards
Mischa

**Comment by Guilouz** [comment]
I have planned a big update of config files and klipperscreen this week-end with ability to calibrate z offset
